### PR TITLE
feat : update all packages of base debian image

### LINKED ISSUE #2073: Do package definition update on base images before building image to reduce Vulnerability
Is this a bug report or feature request?
* Feature Request

**Feature Request**

**What should the feature do:**

Update the package definition update on base images before building image to install latest available package.

**What would be solved through this feature:**

Updating and installing latest package will help to reduce Vulnerability

**Does this have an impact on existing features:**

This shouldn't as ceph related package will install their required dependencies.

Ceph tracker https://tracker.ceph.com/issues/57933

### PR DESCRIPTION
**Pull request by Rajpratik71**
Description of your changes:

Update all package on base images before building image to install latest available package. Updating and installing latest package will help to reduce Vulnerability.

Which issue is resolved by this Pull Request:
Resolves #2073

Signed-off-by: Pratik Raj <rajpratik71@gmail.com>

Checklist:
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.

### CODE REVIEW

File: ceph-releases/ALL/debian/daemon/__DOCKERFILE_INSTALL__ [hunk 1]
```diff
@@ -1,4 +1,5 @@
 echo 'Install packages' && \
+      DEBIAN_FRONTEND=noninteractive apt-get dist-upgrade && \
```
Comment (mohag) [inline ceph-releases/ALL/debian/daemon/__DOCKERFILE_INSTALL__:2]: This should probably be after the `apt-get update` that that can fetch an up to date list if the available packages

Comment (Rajpratik71) [reply ceph-releases/ALL/debian/daemon/__DOCKERFILE_INSTALL__]: ohh yes. Fixed.

**Review by mohag — APPROVED** [review]
The order looks better to me.

**Comment by guits** [comment]
jenkins lint

**Comment by guits** [comment]
jenkins flake8

**Review by guits — APPROVED** [review]

**Pull request merged**